plugin/package.json

```diff
@@ -12,10 +12,10 @@
     "start:datacatalog": "vite -c vite.config.datacatalog-web.ts",
     "start:sidebar": "vite -c vite.config.sidebar-web.ts",
     "build": "run-s typecheck build:web build:plugin",
-    "build:plugin": "run-s build:sidebar build:datacatalog",
+    "build:plugin": "run-s build:sidebar build:datacatalog  ",
```

Comment:
Be careful when changing the package json since we don't lint it. Make sure any spaces in this file are removed.